<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Work</class>
 <widget class="QWidget" name="Work">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>780</width>
    <height>520</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="label_left">
   <property name="geometry">
    <rect>
     <x>-1</x>
     <y>0</y>
     <width>140</width>
     <height>520</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: qlineargradient(spread:pad, x1:0.512438, y1:0.159091, x2:0.497512, y2:0.971591, stop:0.19403 rgba(113,109,255,1), stop:1 rgba(164,125,255,1));</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_top">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>0</y>
     <width>650</width>
     <height>40</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_18">
   <property name="geometry">
    <rect>
     <x>465</x>
     <y>0</y>
     <width>2</width>
     <height>40</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(238, 238, 238);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_19">
   <property name="geometry">
    <rect>
     <x>681</x>
     <y>0</y>
     <width>2</width>
     <height>40</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(238, 238, 238);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_welcome">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>0</y>
     <width>211</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>USER NAME,welcome back !</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_home">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>94</y>
     <width>101</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 14pt &quot;ProximaNova&quot;;
font-weight: bold;
color: rgb(135, 135, 135);</string>
   </property>
   <property name="text">
    <string>homepage</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_work">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>134</y>
     <width>101</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 14pt &quot;ProximaNova&quot;;
font-weight: bold;
color: rgb(135, 135, 135);</string>
   </property>
   <property name="text">
    <string>workspace</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_user">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>178</y>
     <width>101</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 14pt &quot;ProximaNova&quot;;
font-weight: bold;
color: rgb(135, 135, 135);</string>
   </property>
   <property name="text">
    <string>user</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_sett">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>366</y>
     <width>101</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 14pt &quot;ProximaNova&quot;;
font-weight: bold;
color: rgb(135, 135, 135);</string>
   </property>
   <property name="text">
    <string>settings</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_noti">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>408</y>
     <width>101</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 14pt &quot;ProximaNova&quot;;
font-weight: bold;
color: rgb(135, 135, 135);</string>
   </property>
   <property name="text">
    <string>history</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_sugg">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>450</y>
     <width>101</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 14pt &quot;ProximaNova&quot;;
font-weight: bold;
color: rgb(135, 135, 135);</string>
   </property>
   <property name="text">
    <string>instructions</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>130</y>
     <width>5</width>
     <height>30</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>5</x>
     <y>130</y>
     <width>129</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgba(255, 255, 255, 51);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>40</y>
     <width>651</width>
     <height>481</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(237, 241, 247);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_20">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>50</y>
     <width>391</width>
     <height>301</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/other/photo2.jpg);
border-radius:15px</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_21">
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>50</y>
     <width>211</width>
     <height>361</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);
border-radius:15px</string>
   </property>
   <property name="text">
    <string>选项</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_22">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>370</y>
     <width>391</width>
     <height>141</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);
border-radius:15px</string>
   </property>
   <property name="text">
    <string>表格</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>610</x>
     <y>420</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: qlineargradient(spread:pad, x1:0.52, y1:1, x2:0.54, y2:0, 
stop:0.0112994 rgba(110,102,162,1), 
stop:1 rgba(110,102,162,1));
color: rgb(255, 255, 255, 255);
 
border:0px groove gray;
border-radius:15px;
padding:2px 4px;
font: 18pt &quot;Candara&quot;;

</string>
   </property>
   <property name="text">
    <string>Submit</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_2">
   <property name="geometry">
    <rect>
     <x>610</x>
     <y>470</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: qlineargradient(spread:pad, x1:0.52, y1:1, x2:0.54, y2:0, 
stop:0.0112994 rgba(110,102,162,1), 
stop:1 rgba(110,102,162,1));
color: rgb(255, 255, 255, 255);
 
border:0px groove gray;
border-radius:15px;
padding:2px 4px;
font: 18pt &quot;Candara&quot;;

</string>
   </property>
   <property name="text">
    <string>Withdraw</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_1">
   <property name="geometry">
    <rect>
     <x>7</x>
     <y>86</y>
     <width>31</width>
     <height>31</height>
    </rect>
   </property>
   <property name="mouseTracking">
    <bool>false</bool>
   </property>
   <property name="acceptDrops">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/icon/homepage.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_4">
   <property name="geometry">
    <rect>
     <x>7</x>
     <y>130</y>
     <width>31</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/icon/working.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_3">
   <property name="geometry">
    <rect>
     <x>7</x>
     <y>172</y>
     <width>31</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/icon/user.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_5">
   <property name="geometry">
    <rect>
     <x>7</x>
     <y>360</y>
     <width>31</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">
border-image: url(:/icon/setting.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_6">
   <property name="geometry">
    <rect>
     <x>7</x>
     <y>444</y>
     <width>31</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/icon/notice.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_7">
   <property name="geometry">
    <rect>
     <x>7</x>
     <y>402</y>
     <width>31</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/icon/suggestion.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_portrait">
   <property name="geometry">
    <rect>
     <x>710</x>
     <y>0</y>
     <width>40</width>
     <height>40</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-width: 1px;
border-image: url(:/other/photo.jpg);
border-style: solid;
border-color: rgb(0, 0, 0);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
